lib/cartopy/tests/test_img_tiles.py

```diff
@@ -328,6 +331,81 @@ def test_azuremaps_get_image():
     assert extent1 == extent2
 
 
+@pytest.mark.network
+@pytest.mark.parametrize('cache_dir', ["tmpdir", True, False])
```

Comment:
I think we need to add a skipif OWSLIB to handle it not installed as well since it is an optional dependency.
```suggestion
@pytest.mark.parametrize('cache_dir', ["tmpdir", True, False])
@pytest.mark.skipif(not ogc._OWSLIB_AVAILABLE, reason='OWSLib is unavailable.')
```